math/pronicnumber.go

```diff
@@ -0,0 +1,21 @@
+// pronicnumber.go
+// description: Returns true if the number is pronic and false otherwise
+// pronic: Consisting of a base or root or first power plus a higher power.
```

Comment:
It seems I was mistaken, you had already defined `pronic number` but I had overlooked it in my previous review. Apologies. You can remove the line now 😄

```suggestion
```

or change the `details` block to the definition like so
```suggestion
// Pronic number: For any integer n, if there exists integer m such that n = m * (m + 1) then n is called a pronic number.
```

I ask you to remove it because that sentence doesn't make any sense 😓

Reply:
Fixed this 👍